daemon/endpoint.go

```diff
@@ -600,20 +565,20 @@ func (h *getEndpointIDHealthz) Handle(params GetEndpointIDHealthzParams) middlew
 // The `add` labels take precedence over `del` labels, this means if the same
 // label is set on both `add` and `del`, that specific label will exist in the
 // endpoint's labels.
-func (d *Daemon) UpdateSecLabels(id string, add, del labels.Labels) middleware.Responder {
+func (d *Daemon) UpdateSecLabels(id string, add, del labels.Labels) (int, error) {
 	addLabels, _ := labels.FilterLabels(add)
 	delLabels, _ := labels.FilterLabels(del)
```

Comment:
Previously we just handed back an API error and the caller would send that out. Now, there's another path, being called from within the daemon. In this case it could be handled either inside here or in the caller. However, for the case that this command is triggered from an API call, we still need to forward the error back on out.

I've left this as-is for the moment.